src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/8wq9d0uj3EJp5vK6kI49.png", alt="LCP CWV graph in 'Platform': 16 March to 1 June 2021.", width="800", height="344" %}
+  <figcaption>LCP CWV graph in 'Platform': 16 March to 1 June 2021.</figcaption>
+</figure> 
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/bxANzHIw78TvdS1RFyWG.png", alt="FID CWV graph in 'Platform': 16 March to 1 June 2021.", width="800", height="345" %}
+  <figcaption>FID CWV graph in 'Platform': 16 March to 1 June 2021.</figcaption>
+</figure> 
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/7woJXpiHLZU3l7PCdspi.png", alt="CLS CWV graph in 'Platform': 16 March to 1 June 2021.", width="800", height="344" %}
+  <figcaption>CLS CWV graph in 'Platform': 16 March to 1 June 2021.</figcaption>
+</figure> 
+
+CWV results obtained using the "Platform" are in line with the growth of metric values in [Chrome UX Report (CrUX)](https://developers.google.com/web/tools/chrome-user-experience-report).
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/41Czys52EyEMyYv3fI2w.png", alt="LCP metric change in CrUX in 2021.", width="800", height="499" %}
+  <figcaption>LCP metric change in CrUX in 2021.</figcaption>
+</figure> 
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/c8LWvsC52Q58uvKOIZXc.png", alt="FID metric change in CrUX in 2021.", width="800", height="498" %}
+  <figcaption>FID metric change in CrUX in 2021.</figcaption>
+</figure> 
+
+<figure class="w-figure">
+  {% Img src="image/dB6B4Sr8kaaT0KZujRBFC303oFR2/ilAn6vKt93uh4ioqGKj1.png", alt="CLS metric change in CrUX in 2021.", width="800", height="492" %}
+  <figcaption>CLS metric change in CrUX in 2021.</figcaption>
+</figure>
+ 
+A comparison of mean user session duration values a week before the roll-out of initial improvements and a week after the roll-out shows 2.7% growth. Moreover, there is an overall significant increase in conversion in most sections of the page. In particular, conversions to the Mail.ru email app increased by 11.6%, the conversion of the news section increased by 13.5%.
+ 
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">181<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Boost of share of good CLS threshold</p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">2.7<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Higher mean session duration</p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">13.5<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Increase of news section conversion rate</p>
+  </div>
+</div>
+ 
+The most unexpected result we got was a 17.4% increase in the Click-Through Rate (CTR) of the marketing banner (its rendering time was significantly reduced by the introduction of SSR and preload tags).
+ 
+{% Aside 'caution' %}
```

Comment:
```suggestion
{% Aside %}
```

Reply:
Not sure this rises to the level of a `'caution'`, more that it's explaining an unintended side effect of the optimizations you made. But it definitely should still be in an aside!